ds-collector/ds-collector

```diff
@@ -811,6 +814,8 @@ read_config() {
     statusState=$?
     copyConfig=true
     collectionArgs+=(-f $baseDir/$configFile)
+    # change the $skipNodes string into an array
+    IFS=' ' read -r -a skipNodes <<< "$skipNodes"
```

Comment:
The separator is defined as a space, but the config file doesn't mention if it should be a space separated list of nodes, or a comma separated list of nodes. Probably worth making clear.
Is there a reason to have the IFS and read command on the same line?
using -r , backslash is treated as literal, not an escape - was there a reason for this being added to the params?

Reply:
> The separator is defined as a space, but the config file doesn't mention if it should be a space separated list of nodes, or a comma separated list of nodes. 
documented.

> Is there a reason to have the IFS and read command on the same line?
yes. scope.

> using -r , backslash is treated as literal, not an escape - was there a reason for this being added to the params?
nope. removed.

Reply:
`-a` is not compatible on macos.